crates/libtiny_tui/src/key_map.rs

```diff
@@ -357,13 +359,19 @@ fn deser_keymap() {
       tab_goto: 1
     a: 
       input: b
+    b:
+      run_irc_command: clear
```

Comment:
I think this may be confused with actual IRC commands like PRIVMSG, QUIT, JOIN etc.

I suggest just "command".

Reply:
Right. Will fix.